elasticsearch/src/main/java/org/hibernate/search/elasticsearch/client/impl/DefaultElasticsearchClient.java

```diff
@@ -59,6 +67,10 @@ public Response execute(ElasticsearchRequest request) throws IOException {
 			 */
 			return e.getResponse();
 		}
+		finally {
+			long executionTime = System.currentTimeMillis() - start;
```

Comment:
To measure durations don't use `System.currentTimeMillis()`, use System.nanoTime()